docs/directory-api-reference/graphql.mdx

```diff
@@ -0,0 +1,200 @@
+---
+sidebar_label: GraphQL
+title: Directory Guide
+description: How to use the Aserto Directory GraphQL API
+---
+
+# GraphQL APIs
+
+## Schema Documentation
+
+You can view the Directory GraphQL API documentation in the [GraphQL Playground](https://directory.eng.aserto.com/graph#). If you have an Aserto account, you can also access the Playground from within the [Aserto Console](../aserto-console/directory/graphql-playground)
+
+### Schema documentation
+
+Launch the stand-alone [Playground](https://directory.eng.aserto.com/graph#). This is ideal for quickly exploring the API documentation.
```

Comment:
we should never use `.eng` urls - should be https://directory.prod.aserto.com/graph